models/practice.py

```diff
@@ -26,6 +26,17 @@
 
 db.define_table('user_topic_practice_Completion',
                 Field('user_id', db.auth_user),
-                Field('course_name','string'),
+                Field('course_name', 'string'),
                 Field('practice_completion_time', type='date'),
                 migrate='user_topic_practice_Completion.table')
+
+
+# db.define_table('user_topic_practice_survey',
+#                 Field('user_id', db.auth_user,
+#                       default=auth.user_id, update=auth.user_id, writable=False),
+#                 Field('course_name', 'string'),
+#                 Field('like_practice', requires=IS_IN_SET(['Like', 'Dislike', 'No response'])),
+#                 Field('feedback', 'text'),

```

Comment:
Please remove this commented code. You put the table in a models file, I presume.